web/src/containers/AssurancesAndCompliance.js

```diff
@@ -96,6 +96,8 @@ class AssurancesAndCompliance extends Component {
                           hideLabel
                           value={explanation}
                           onChange={this.handleExplanationChange(name, index)}
+                          className="m0 textarea textarea-sm"
```

Comment:
I don't entirely understand [higher-order components in React](https://reactjs.org/docs/higher-order-components.html), but we're using them to render textareas. Somehow textareas seem to magically get `m0 textarea` classes, and when I added `textarea-sm`, it overwrote those classes. So I feel like I'm probably duplicating these from somewhere, but I can't figure out where they're coming from.

Reply:
The `mo textarea` classes comes from `components/Inputs/index.js`.  It then gets wrapped with an `InputWrapper`, which applies a label around the input.  The wrapper also takes every prop it doesn't know and applies them directly to the input element.  In this case, the wrapper doesn't know about `className`, so it just passes it along, thus overriding the class set initially.

Is it useful to set the `m0 textarea` class on every textarea, or would it be better to target them with CSS?  Then we could ditch the hard-coded `m0 textarea` and using the `className` prop would work as expected?

Reply:
Hm, so, I think we're going to be revisiting what spacing units we use in #1117, so maybe it makes sense to deal with this then? (Applying the spacing class to every element is the correct Atomic Way to do this, I believe, but this might be one of the instances in which we want to use fewer classes with more rules each in order to consistently apply spacing, depending on what we figure out in that issue.)